pages/community/dei-mission-statement.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: DEI Mission Statement
+subtitle: Diversity, Equality, and Inclusion at US-RSE
+bigimg:
+permalink: /dei-mission/
+---
+
+We believe Diversity, Equity, and Inclusion (DEI) are core values of the
+US-RSE Association. A dedicated team of active members continue to lead the
+development of a DEI program for our organization. DEI discussions are and
+will continue to be an integral part of our community.
+
+The US-RSE Association is committed to providing an [inclusive environment with
+equitable treatment]({{site.url}}/code-of-conduct) for all and to promoting and
```

Comment:
```suggestion
equitable treatment]({{site.url}}/code-of-conduct/) for all and to promoting and
```

Reply:
You know I originally had this, but then looked at some other pages and removed it to follow the others. I'll add the `/` to the ends of the links.